dist/designer/xoom

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
-java -jar $VLINGO_XOOM_DESIGNER_HOME/bin/xoom-designer-dist.jar $@ --currentDirectory ${PWD}
+export VLINGO_XOOM_STARTER_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
You brought back the old environment name. It should be `VLINGO_XOOM_DESIGNER_HOME` instead of `VLINGO_XOOM_STARTER_HOME`.

Reply:
corrected